src/App.js

```diff
@@ -0,0 +1,25 @@
+import logo from './logo.svg';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <img src={logo} className="App-logo" alt="logo" />
+        <p>
+          Edit <code>src/App.js</code> and save to reload.
+        </p>
+        <a
+          className="App-link"
+          href="https://reactjs.org"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          Learn React
+        </a>
+      </header>
+    </div>
+  );
+}
+
+export default App;
```

Comment:
There's nothing here but boilerplate code for `create-react-app`.

![image](https://user-images.githubusercontent.com/38088818/201467188-34b9f70b-33f8-4612-a0af-b99d0a864821.png)

For the front end React app should have the following:

- The `App` component should have a route (from react-router) rendering the `Greeting` component.
- The should be a `Greeting` component under the directory `components`.
- The should be a Redux store, an action, and a reducer that uses the URL (can be the localhost URL since the API will be run locally) to the endpoint from the Rails API
- The two apps should work together:
   * When the rails API is run locally, the React app uses the localhost URL and displays a random greeting from the API